src/test/java/org/jvnet/hudson/test/recipes/LocalDataExtendedTest.java

```diff
@@ -0,0 +1,8 @@
+package org.jvnet.hudson.test.recipes;
+
+
+/**
+ * Illustrates a case where a test class extends another test class that has test methods using the LocalData recipe.
+ */
+public class LocalDataExtendedTest extends LocalDataZipTest {
```

Comment:
just to confirm this is intended not to have any test inside it and it fails without the change in `src/main`?

Reply:
Yes, the purpose is for it to pick up the test method defined in `LocalDataZipTest`.